content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/cycle_counts.rst

```diff
@@ -2,40 +2,122 @@
+
+Change inventory count frequency by location
+============================================
 
-Change the inventory frequency
-==============================
+Now that the storage locations setting is enabled, the inventory count frequency can be changed for
+specific locations created in the warheouse.
 
-To change a location's inventory frequency, first, go to the locations by clicking
-:menuselection:`Inventory --> Configuration --> Locations`.
+To view (and edit) locations, navigate to :menuselection:`Inventory app --> Configuration -->
+Locations`. This reveals a :menuselection:`Locations` page containing every location currently
+created and listed in the warehouse.
 
-Then, click on a location to open the location settings. Next, click on :guilabel:`Edit`. In the
-:guilabel:`Inventory Frequency (Days)` field, set the number of days. For example, a location that
-needs an inventory count every 30 days would set the :guilabel:`Inventory Frequency (Days)` value
-to `30`. Once the value is entered, click :guilabel:`Save` to apply the setting to the location.
-Now, once an inventory adjustment is applied to this location, the next scheduled count date will
-be automatically set based on the number of days in the :guilabel:`Inventory Frequency (Days)`
-setting.
+From this page, click into a location to reveal the settings and configuration page for that
+location. Click :guilabel:`Edit` to edit the location settings.
 
-.. image:: cycle_counts/inventory-frequency.png
+Under the :guilabel:`Cyclic Inventory` section, locate the :guilabel:`Inventory Frequency (Days)`
+field, which should be set to `0` (if this location has not been edited previously). In this field,
+change the value to whichever number of days is desired.
+
+.. image:: cycle_counts/cycle-counts-location-frequency.png
    :align: center
-   :alt: Edit a location to change the inventory frequency.
+   :alt: Location frequency setting on location.
+
+.. example::
+   A location that needs an inventory count every 30 days should have the :guilabel:`Inventory
+   Frequency (Days)` value set to `30`.
+
+Once the frequency has been changed to the desired number of days, click :guilabel:`Save` to save
+changes. Now, once an inventory adjustment is applied to this location, the next scheduled count
+date will be automatically set based on the value entered into the :guilabel:`Inventory Frequency
+(Days)` field.
+
+Count inventory by location
+===========================
+
+To perform a cycle count for a specific location in the warehouse, navigate to
+:menuselection:`Inventory app --> Operations --> Inventory Adjustments`. This reveals an
+:guilabel:`Inventory Adjustments` page containing all products currently in stock, with each product
+listed on its own line.
+
+From this page, the :guilabel:`Filters` and :guilabel:`Group By` buttons (at the top of the page,
+under the :guilabel:`search bar`), can be used to select specific locations and perform inventory
+counts.
```

Comment:
```suggestion
under the :guilabel:`Search...` bar), can be used to select specific locations and perform
inventory counts.
```
i know that there's been some back and forth re: the search bar, and if it should have a guilabel. I think, if we're going to use one, we should be as accurate as possible and convey *exactly* what's seen on the screen - hence my suggestion...what do you think? 🤔